.github/workflows/ci.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
     needs: lint
     strategy:
       matrix:
-        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
+        python-version: ["3.10", "3.11", "3.12", "3.13"]
```

Comment:
Minor: could you please add 3.14 to the matrix?

Reply:
sure, done